README.md

```diff
@@ -489,6 +489,31 @@ $ git push heroku master
 If the app is deployed successfully, congratulations! You've finished setting
 up, deploying to, and hosting your bot on Heroku!
 
+## Troubleshooting
+
+Sometimes mistakes happen and like with most programming related problem - the solution is reading logs and documentation. 
+To check your Heroku app logs you can use command `heroku logs --tail` and read what might be cause of your problem.
+
+In case you see following line in your logs:
+
+`Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch`
+
+Then you should check if you are running your app correct dyno - It should running it on worker and not on web, 
```

Comment:
See my previous comment about rephrasing; the dash isn't correct punctuation, "It should running" isn't proper grammar, "dyons" is misspelled, etc.